### DISCUSSION TITLE
How to avoid reaching out to other API's (googleapi for instance)?

### DISCUSSION THREAD

**Discussion by ericbugin** [keras-team/autokeras#1511] [Q&A]
I am on an environment where I cannot access normal sites such as googleapis. If I run the autokeras ImageClassifier with max_trails=1 or 2, it's fine, but if I set it to 3 or > it begins to reach out to googleapis to download the algorithm used for the 3rd trial (from my understanding), and breaks since I can't reach the site. 

My desire is to avoid using algorithms that require downloading form external sources while using as many algorithms as possible (or at least greater than 2). 

Is there a way to selectively avoid using certain trials?

**Comment by haifeng-jin** [comment]
Currently, there is no such way to do so.
The only way I can think of is to avoid those blocks by building your own search space.
https://autokeras.com/tutorial/customized/
